### PR TITLE
Adds api key in client/env/config.js file & adds this file name to .gitignore

### PR DESCRIPTION
**Pull request by EricMFischer**

### CODE REVIEW

**Comment by ntoscano** [comment]
+1

**Comment by cassiebeckley** [comment]
You might want to add an example config file - something like client/env/config.js.example with a placeholder for the key.

**Comment by hsadler** [comment]
Sorry, there is a merge conflict. Please rebase and push again. Thanks!

File: server/api-config.js [hunk 1]
```diff
@@ -0,0 +1 @@
+exports.apiKey = 'NNfUcLio9cmshd6YN03HGDXOuYBnp1VaKpqjsnDdLi4uKD11Ny';
```
Comment (cassiebeckley) [inline server/api-config.js]: You'll probably want to remove this file.

**Comment by ntoscano** [comment]
+1

**Comment by hsadler** [comment]
+1

**Comment by cassiebeckley** [comment]
:+1:

**Pull request merged**